cwms_radar_api/src/main/java/cwms/radar/datasource/DelegatingConnectionPreparer.java

```diff
@@ -28,6 +38,7 @@ public DelegatingConnectionPreparer(ConnectionPreparer prep1, ConnectionPreparer
     public Connection prepare(Connection connection) throws SQLException {
         Connection retval = connection;
         for (ConnectionPreparer delegate : delegates) {
+            logger.info(delegate.getClass().getName());
```

Comment:
I think this info() statement should be something quieter.

Reply:
Yep. Missed one. Thanks.